api/src/services/stardust/chronicleService.ts

```diff
@@ -158,12 +157,12 @@ export class ChronicleService {
     /**
      * Download the transaction history of an address.
      * @param address The address to download history for.
-     * @param timespan The timespan to use.
+     * @param targetDate The date to use.
      * @returns The history reponse.
      */
     public async transactionHistoryDownload(
         address: string,
-        timespan: TimespanOption
+        targetDate: moment.Moment
```

Comment:
Maybe we can accept targetDate as string instead of moment.Moment and than use the moment lib to get the date.